packages/module/src/controller/QuickStartTaskHeader.tsx

```diff
@@ -69,39 +69,31 @@ const QuickStartTaskHeader: React.FC<QuickStartTaskHeaderProps> = ({
   const notCompleted = taskStatus === QuickStartTaskStatus.VISITED;
   const skippedReviewOrFailed =
     taskStatus === QuickStartTaskStatus.REVIEW || taskStatus === QuickStartTaskStatus.FAILED;
+
   const tryAgain = !isActiveTask && (skippedReviewOrFailed || notCompleted) && (
-    <FlexItem>
-      <Title headingLevel="h4" className="pfext-quick-start-task-header__tryagain">
-        Try the steps again.
-      </Title>
-    </FlexItem>
+    <>
+      <div />
```

Comment:
why an empty div? if it's to create space, maybe add it to the `pfext-quick-start-task-header__tryagain` class, and since it'll just be one element we also won't need the fragment

Reply:
The extra div here is because I'm adding an extra row to the grid layout in the `.pfext-quick-start-task-header` parent. I to make this more explicit by using a PF `Grid` but the `grid-template-colums` value I needed was not available.